### PR TITLE
Markdown improvements

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
* Puntuaction in change log for readability.
* ~Upgrade code of conduct to its latest version.~ Deferred to a further PR
* Fix some markdownlint issues.
* Enforce a very relaxed line length limit in markdown files.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,22 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at deivid.rodriguez@riseup.net. All
```
Comment (varyonic) [inline CODE_OF_CONDUCT.md]: You are responsible for enforcement?

Comment (deivid-rodriguez) [reply CODE_OF_CONDUCT.md]: Which email address can we use? I'd like all of us helping enforcing it, but I don't mind using my email and then forwarding anything to the other maintainers.

Comment (deivid-rodriguez) [reply CODE_OF_CONDUCT.md]: I also thought about telling users to contact any maintainer privately through slack, but I'm not sure slack is as universally used / available as email is...

Comment (varyonic) [reply CODE_OF_CONDUCT.md]: Typically you could setup email forwarding at the domain provider but I think only Greg has that access for activeadmin.info.

Comment (deivid-rodriguez) [reply CODE_OF_CONDUCT.md]: Ha, funny I had read only the previous version of your comment but decided to do just what you asked for. It was not really related anyways :)

**Pull request merged**